.github/workflows/sonarcloud-web.yaml

```diff
@@ -0,0 +1,22 @@
+name: SonarCloud Web
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  workflow_call:
+    secrets:
+        SONAR_TOKEN_WEB:
+          required: true
+  workflow_dispatch: ~
+jobs:
+  sonarcloud:
+    name: SonarCloud
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+      - name: SonarCloud Scan
+        uses: SonarSource/sonarcloud-github-action@master
```

Comment:
Please check the folder where runs the step, it should be ./src/App.UI

Reply:
Done